pkg/filesystem/fs.go

```diff
@@ -524,3 +623,90 @@ func (fs *Filesystem) createDaemon(mountpoint string, ref int32) (d *daemon.Daem
 func (fs *Filesystem) DaemonBacked() bool {
 	return config.GetDaemonMode() != config.DaemonModeNone
 }
+
+func (fs *Filesystem) getManager(fsDriver string) (*manager.Manager, error) {
+	switch fsDriver {
+	case config.FsDriverBlockdev:
```

Comment:
It seems the codes will be hard to maintain once we add a new fs driver.

Reply:
Any suggestion here?

Reply:
Maybe use a `ManagerCoordinator` to manage these `xxxManager`.